src/chartsFactory.jsx

```diff
@@ -61,7 +61,13 @@ module.exports = function (chartType, Highcharts){
     },
 
     render: function (){
-      let props = this.props;
+      let {
+          callback: callbackIgnored,
+          config: configIgnored,
+          isPureConfig: isPureConfigIgnored,
+          neverReflow: neverReflowIgnored,
```

Comment:
Can you ret rid of the right (...Ignored) part, so it's just `..., callback, config, isPureConfig, ...`